packages/playground/src/dashboard/transfer_view.vue

```diff
@@ -232,7 +232,6 @@ async function submitFormTwinID() {
     if (twinDetails != null) {
       loadingTwinIDTransfer.value = true;
       await transfer(twinDetails);
-      loadingTwinIDTransfer.value = false;
```

Comment:
is this related ?

Reply:
no i returned it, it was by mistake when i was merging conflicts.